src/test/java/com/linecorp/armeria/server/docs/DocServiceTest.java

```diff
@@ -52,17 +53,24 @@
     @Override
     protected void configureServer(ServerBuilder sb) {
         final ThriftService helloService = new ThriftService(HELLO_SERVICE_HANDLER);
-        final ThriftService fooService = new ThriftService(mock(FooService.AsyncIface.class));
-        final ThriftService cassandraService = new ThriftService(mock(Cassandra.AsyncIface.class));
+        final ThriftService fooService = new ThriftService(mock(FooService.AsyncIface.class),
+                                                           SerializationFormat.THRIFT_BINARY,
+                                                           EnumSet.of(SerializationFormat.THRIFT_BINARY));
```

Comment:
The default behavior of ThriftService is to allow all formats even when specifying a default, this is just testing the probably non-standard case of explicitly restricting the allowed formats to not include TTEXT. We can either default allowed to match default, or allowed to be all and I find defaulting to all to be easier to use in common use cases.